### PR TITLE
Doc(EJ2-849633): UG for OnDemand Checkbox/Excel Filter type

### PR DESCRIPTION
**Pull request by vasanthakumarsf3389**
Ug for Excel or Checkbox filtering with on-demand feature

### CODE REVIEW

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/filtering/excel-like-filter.md [hunk 1]
```diff
@@ -36,6 +36,57 @@ You can enable Excel like filter by defining. [`type`](https://help.syncfusion.c
...
+
+The Excel/Checkbox filter retrieves distinct data in ascending order, governed by its [itemsCount](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Grids.GridFilterSettings.html#Syncfusion_EJ2_Grids_GridFilterSettings_ItemsCount) property, with a default value of **50**. As the checkbox list data scroller reaches its end, the next dataset is fetched and displayed, with the notable advantage that this process only requests new checkbox list data without redundantly fetching the existing loaded dataset.
+
+### Customize the items count for initial rendering
+
+Based on the items count value, the Excel/Checkbox filter gets unique data and displayed in Excel/Checkbox filter content dialog. You can customize the count of on-demand data rendering for Excel/Checkbox filter by adjusting the [ItemsCount](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Grids.GridFilterSettings.html#Syncfusion_EJ2_Grids_GridFilterSettings_ItemsCount) property. The default value is `50`.
+
+```html
+<e-grid-filterSettings enableInfiniteScrolling="true" itemsCount="40"></e-grid-filterSettings>
```
Comment (MaithiliyK) [inline ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/filtering/excel-like-filter.md:52]: is the snippet is in core format?

**Review by MaithiliyK — COMMENTED** [review]
address my review comments

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/filtering/excel-like-filter.md (hunk 1 above)
Comment (vasanthakumarsf3389) [reply ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/filtering/excel-like-filter.md:52]: yes, this is core format, ensured by running local sample

**Review by MaithiliyK — APPROVED** [review]
changes are fine

**Pull request merged**